src/operator/account/MigrateOrgs.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  AlignItems,
+  Button,
+  ButtonType,
+  ComponentColor,
+  ComponentSize,
+  ComponentStatus,
+  FlexBox,
+  FlexDirection,
+  Form,
+  Input,
+} from '@influxdata/clockface'
+import React, {ChangeEvent, FC, useContext, useState} from 'react'
+import {AccountContext} from '../context/account'
+import MigrateOrgsOverlay from './MigrateOrgsOverlay'
+import {OperatorAccount, getOperatorAccount} from 'src/client/unityRoutes'
+
+const MigrateOrgsTool: FC = () => {
+  const {account, setMigrateOverlayVisible} = useContext(AccountContext)
+  const [toAccountId, setToAccountId] = useState('')
+  const [toAccount, setToAccount] = useState<OperatorAccount>(null)
+
+  const changeToAccountId = (event: ChangeEvent<HTMLInputElement>) => {
+    setToAccountId(event.target.value)
+  }
+
+  const submit = async () => {
+    if (toAccountId === '') {
+      return
```

Comment:
Do we need a notification here - like "please enter an account to migrate to"?

Reply:
Yeah, that's a good point. This is a pretty blind failure. It seems like I can pull use the `useDispatch()` hook with `notify` to put up an error. Does that make sense?

Also, remocal is failing me... so I'm pushing up these changes to run in CI 😅 🤞